doc/ocpp_201_status.md

```diff
@@ -1692,27 +1692,27 @@ This document contains the status of which OCPP 2.0.1 numbered functional requir
 
 ## Diagnostics - Set Variable Monitoring
 
-| ID        | Status | Remark |
-|-----------|--------|--------|
-| N04.FR.01 | ✅     |        |
-| N04.FR.02 | ✅     |        |
-| N04.FR.03 | ✅     |        |
-| N04.FR.04 | ✅     |        |
-| N04.FR.05 |        |        |
-| N04.FR.06 | ✅     |        |
-| N04.FR.07 | ✅     |        |
-| N04.FR.08 | ✅     |        |
-| N04.FR.09 | ✅     |        |
-| N04.FR.10 | ✅     |        |
-| N04.FR.11 | ✅     |        |
-| N04.FR.12 | ✅     |        |
-| N04.FR.13 | ✅     |        |
-| N04.FR.14 | ✅     |        |
-| N04.FR.15 |        |        |
-| N04.FR.16 | ✅     |        |
-| N04.FR.17 |        |        |
-| N04.FR.18 | ✅     |        |
-| N04.FR.19 | ✅     |        |
+| ID        | Status | Remark                                   |
+|-----------|--------|------------------------------------------|
+| N04.FR.01 | ✅     |                                          |
+| N04.FR.02 | ✅     |                                          |
+| N04.FR.03 | ✅     |                                          |
+| N04.FR.04 | ✅     |                                          |
+| N04.FR.05 | ❎     | Everything is supported on our charger   |
+| N04.FR.06 | ✅     |                                          |
+| N04.FR.07 | ✅     |                                          |
+| N04.FR.08 | ✅     |                                          |
+| N04.FR.09 | ✅     |                                          |
+| N04.FR.10 | ✅     |                                          |
+| N04.FR.11 | ✅     |                                          |
+| N04.FR.12 | ✅     |                                          |
+| N04.FR.13 | ✅     |                                          |
+| N04.FR.14 | ✅     |                                          |
```

Comment:
PR description says that this is not fullfiled

Reply:
Is is 'N04.FR.15' which is not supported. 'N04.FR.14' is supported and tested via OCTT.